### DISCUSSION TITLE
Self-hosting advice

### DISCUSSION THREAD

**Discussion by Mubelotix** [placeAtlas/atlas-2022#1506] [General]
I just ran an r/place-like event in my small community. Is there a guide somewhere that explains how to change the canvas data? I have been unable to quickly locate the data files

**Comment by Mubelotix** [comment]
I have found [this commit](https://sc07.dev/marius851000/fediverse-canvas-atlas-2024/-/commit/3a212f12523d11f79fe9376808a95fe5e7af2f1a) to help, but I'm still open to advices

**Reply by Xeravax** [reply]
atlas.json contains all the data, we use a the merge_out script to merge all changes from a json file made through the web editor into a single (compressed) file

**Reply by Xeravax** [reply]
Like so: https://2023.place-atlas.stefanocoding.me/atlas.json
